app/templates/teacherhome/liveformlectures.html

```diff
@@ -27,13 +26,10 @@
                 <a class="nav-link" href="{{url_for('teacherhome_blueprint.pass_grades', course_id = course.id)}}">Grades</a>
             </li>
         </ul>
-    </div>
-
-   
+    </div>  
     <div class="col-md-9">
```

Comment:
Potential improvement:•	Add Comments. Comments should be always in code both back end and front end , so that developers can read and understand the thought process in the code easier.